Trigger a build

milljm opened the pull request:
DO NOT MERGE

Refs #000

moosebuild commented:
Job [WIP Dynamic Modules Darwin](https://civet.inl.gov/job/735909/) on 975e9d5 : invalidated by @milljm

moosebuild commented:
Job [Doxygen](https://civet.inl.gov/job/755887/) on 975e9d5 wanted to post the following:

View the modules doxygen [here](https://mooseframework.inl.gov/docs/PRs/17700/doxygen/modules).

This comment will be updated on new commits.

aeslaughter committed:
Fix CIVET result links for master branch documentation

The fix in #17178 was operting for the PR/devel levels but not linking to the correct pages on
master. This latest change will result in the CIVET badges only being available on the master
documentation. This is the correct behavior, because that is the documentation intented for public
consumption.

(refs #17177)

aeslaughter committed:
For consistency change is_git_repo to git_is_repo

aeslaughter committed:
Only include CIVET results when branch and author match

loganharbour committed:
Remove non *.hpc.inl.gov domains from INL HPC proxy

refs #18070

aeslaughter committed:
Use commands that work for older versions of git

Commit:
Triggering a build

Testing ./configure, make, make install

Refs #000

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/758886/) on 87da664 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/17700/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Code coverage](https://civet.inl.gov/job/758903/) on 87da664 wanted to post the following:

Generated code coverage:<br><br>[moose](https://mooseframework.inl.gov/docs/PRs/17700/coverage/moose)<br>[chemical_reactions](https://mooseframework.inl.gov/docs/PRs/17700/coverage/chemical_reactions)<br>[combined](https://mooseframework.inl.gov/docs/PRs/17700/coverage/combined)<br>[contact](https://mooseframework.inl.gov/docs/PRs/17700/coverage/contact)<br>[external_petsc_solver](https://mooseframework.inl.gov/docs/PRs/17700/coverage/external_petsc_solver)<br>[fluid_properties](https://mooseframework.inl.gov/docs/PRs/17700/coverage/fluid_properties)<br>[fsi](https://mooseframework.inl.gov/docs/PRs/17700/coverage/fsi)<br>[functional_expansion_tools](https://mooseframework.inl.gov/docs/PRs/17700/coverage/functional_expansion_tools)<br>[geochemistry](https://mooseframework.inl.gov/docs/PRs/17700/coverage/geochemistry)<br>[heat_conduction](https://mooseframework.inl.gov/docs/PRs/17700/coverage/heat_conduction)<br>[level_set](https://mooseframework.inl.gov/docs/PRs/17700/coverage/level_set)<br>[misc](https://mooseframework.inl.gov/docs/PRs/17700/coverage/misc)<br>[navier_stokes](https://mooseframework.inl.gov/docs/PRs/17700/coverage/navier_stokes)<br>[peridynamics](https://mooseframework.inl.gov/docs/PRs/17700/coverage/peridynamics)<br>[phase_field](https://mooseframework.inl.gov/docs/PRs/17700/coverage/phase_field)<br>[porous_flow](https://mooseframework.inl.gov/docs/PRs/17700/coverage/porous_flow)<br>[ray_tracing](https://mooseframework.inl.gov/docs/PRs/17700/coverage/ray_tracing)<br>[rdg](https://mooseframework.inl.gov/docs/PRs/17700/coverage/rdg)<br>[richards](https://mooseframework.inl.gov/docs/PRs/17700/coverage/richards)<br>[stochastic_tools](https://mooseframework.inl.gov/docs/PRs/17700/coverage/stochastic_tools)<br>[tensor_mechanics](https://mooseframework.inl.gov/docs/PRs/17700/coverage/tensor_mechanics)<br>[xfem](https://mooseframework.inl.gov/docs/PRs/17700/coverage/xfem)<br>

This comment will be updated on new commits.

Commit:
Trigger a build

Testing make install on meowbuild

Commit:
trigger build

New make install recipes added. Triggering a build

Commit:
New recipes did not populate... try again

moosebuild commented:
All [jobs](https://civet.inl.gov/event/69479/) on 607d347 : canceled by @milljm

milljm commented:
Closing because Civet has decided to not populate newly created recipes... Will open another Issue/PR.
